app/settings.py

```diff
@@ -77,8 +77,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": environ.get("DB_ENGINE"),
+        "NAME": environ.get("DB_NAME"),
+        "USER": environ.get("DB_USER"),
+        "PASSWORD": environ.get("DB_PASSWORD"),
+        "HOST": environ.get("DB_HOST"),
+        "PORT": environ.get("DB_PORT"),
```

Comment:
@gregory-schiano and @weiiwang01 here is an example of what the DB config for Django would be

Reply:
Well this is not necessarily conventional but more "manually" defined here. Django doesn't have any native environment variable setting convention, we'll have to define ours

Reply:
Indeed. It we decide on another approach, I'll open a follow up PR